esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -1913,16 +1915,19 @@ void WsWuInfo::getWorkunitEclAgentLog(const char* fileName, const char* agentPid
                 buf.clear();
             }
 
-            //If we spot the workunit id anywhere in the tacing for this pid then assume it is the correct instance.
+            //If we spot the workunit id anywhere in the tracing for this pid then assume it is the correct instance.
             if(!wuidFound && strstr(line.str(), wuid.str()))
+            {
+                pidPtr = pPid;//remember offset of PID within line
```

Comment:
Good point about the buffer realloc, will refactor to make that safer
The columns are fixed width, not sure if that can be overridden